saleor/graphql/shop/schema.py

```diff
@@ -25,6 +25,9 @@ class ShopQueries(graphene.ObjectType):
     order_settings = graphene.Field(
         OrderSettings, description="Order related settings from site settings."
     )
+    gift_card_settings = graphene.Field(
+        GiftCardSettings, description="Gift card related settings from site settings."
+    )
```

Comment:
Shouldn't be required=True? I think this object will be always returned, but it may have empty fields inside if I'm correct?